Client/mods/deathmatch/logic/lua/CLuaManager.cpp

```diff
@@ -130,23 +128,25 @@ CLuaMain* CLuaManager::GetVirtualMachine(lua_State* luaVM)
     }
 
     // Find a matching VM in our map
-    CLuaMain* pLuaMain = MapFindRef(m_VirtualMachineMap, luaVM);
-    if (pLuaMain)
-        return pLuaMain;
+    {
+        CLuaMain* pLuaMain = MapFindRef(m_VirtualMachineMap, luaVM);
+        if (pLuaMain)
+            return pLuaMain;
+    }
```

Comment:
```suggestion
    CLuaMain* pLuaMain = MapFindRef(m_VirtualMachineMap, luaVM);
    if (pLuaMain)
        return pLuaMain;
```
Is the change in indentation+scoping necessary here?